package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Ensure that your package.json file is in the same directory as your server.js file. Otherwise your files wont be able to find its dependancies.